src/webportal/src/app/vc/vc-modal-component.ejs

```diff
@@ -0,0 +1,61 @@
+<!--  Modal component for vc add -->
+  <div class="modal fade vc-model" id="virtualClustersList" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+     <div class="modal-dialog">
+       <div class="modal-content">
+         <div class="modal-header">
+           <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
+             &times;
+           </button>
+           <h3 class="modal-title">
+             Add New Virtual Cluster
+           </h3>
+           <h5 class="modal-title">The capacity of new virtual cluster will be split from the "default" virtual cluster</h5>
+         </div>
+         <div class="modal-body">
+           <div class="modal-vc-content row">
+             <div class="form-group">
+               <input name="vcname" class="add-vc-fild form-control" placeholder="Virtual Cluster Name">
+             </div>
+             <div class="form-group">
+               <input name="capacity" class="add-vc-fild form-control" placeholder="Virtual Cluster Capacity">
+             </div>
+             <button type="button" class="btn btn-lg btn-primary btn-block" id="virtualClustersListAdd">
+               Add Virtual Cluster
+             </button>
+           </div>
+         </div>
+       </div>
+     </div>
+   </div>
+
+<!-- Modal component for vc edit -->
+<div class="modal fade vc-edit-model" id="virtualClustersEdit" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+   <div class="modal-dialog">
+     <div class="modal-content">
+       <div class="modal-header">
+         <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
+           &times;
+         </button>
+         <h3 class="modal-title">
+           Edit Virtual Cluster
+         </h3>
+       </div>
+       <div class="modal-body">
+         <div class="modal-vc-content">
+           <div class="form-group edit-group">
+             <label for="editName" class="edit-group-item">Name</label>
+             <input id="editName" name="nameEdit" class="add-vc-fild form-control" placeholder="Virtual Cluster Name" readonly=true>
```

Comment:
no `=true` needed in readonly attribute

Reply:
Should this field `nameEdit` be readonly? I didn't find that the [`PUT virtualCluster` API](https://github.com/Microsoft/pai/blob/vc_update/docs/rest-server/API.md#put-virtual-clustersvcname-administrator-only) includes vc name.

Reply:
It is better to set it readonly.